packages/core/src/amazonqFeatureDev/types.ts

```diff
@@ -36,7 +42,7 @@ export enum FollowUpTypes {
     SendFeedback = 'SendFeedback',
 }
 
-export type SessionStatePhase = 'Init' | 'Approach' | 'Codegen'
+export type SessionStatePhase = DevPhase.INIT | DevPhase.APPROACH | DevPhase.CODEGEN
```

Comment:
nit: can we simply make the SessionStatePhase enum.

Reply:
I'll keep a note on that. The intent here is to have type and enums separated. For example, if we want include a new DevPhase that isn't necessarily part of SessionStatePhase. 

However if I notice that this would be exact the same through the weeks, I'll change in the next review. The idea here is to have at least the DevPhase mapped as enums and not hardcoded strings for the moment.

Reply:
> : can we simply make the SessionStatePhase enum.

enums are discouraged in this codebase (they serve no purpose, and are basically deprecated in typescript). Algebraic types as done here are preferred.